test_consume_flow.py::test_compact_publish_and_consume failing sporadically

Update: this test was skipped in https://github.com/oceanprotocol/ocean.py/commit/1bbdef2e4ec157e14bcfb8543ed381c704393145. 

Once it's fixed, un-skip the test.

----

### The Bug

It seems to be sporadic. And fairly recent, first popping up perhaps on Nov 29.

Failure:
```text
ocean_lib.agreements.consumable.AssetNotConsumable: 2
```

Failure log examples: (oldest to newest)
- Merge into main of PR [Towards #1094 - rest of asset -> ddo for ocean_assets.py, more](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189). Note: tests passed in the PR
- PR [Towards #1094 - asset -> ddo, in ocean_compute.py, more](https://github.com/oceanprotocol/ocean.py/actions/runs/3582981516/jobs/6027852372)
- PR [Fix #1136 - Remove stress tests](https://github.com/oceanprotocol/ocean.py/actions/runs/3583249042/jobs/6028453271)

### Traceback

```text
_______________________ test_compact_publish_and_consume _______________________

config = {'ADDRESS_FILE': '~/.ocean/ocean-contracts/artifacts/address.json', 'DOWNLOADS_PATH': 'consume-downloads', 'METADATA_CACHE_URI': 'http://172.15.0.5:5000', 'NETWORK_NAME': 'development', ...}
publisher_wallet = <LocalAccount '0x02354A1F160A3fd7ac8b02ee91F04104440B28E7'>
consumer_wallet = <LocalAccount '0x529043886F21D9bc1AE0feDb751e34265a246e47'>

    @pytest.mark.integration
    def test_compact_publish_and_consume(
        config: dict,
        publisher_wallet,
        consumer_wallet,
    ):
        data_provider = DataServiceProvider
        ocean_assets = OceanAssets(config, data_provider)
    
        # publish
        name = "CEXA"
        url = "https://cexa.oceanprotocol.io/ohlc?exchange=binance&pair=ETH/USDT"
        (data_nft, datatoken, ddo) = ocean_assets.create_url_asset(
            name, url, publisher_wallet
        )
    
        # share access
        datatoken.mint(
            consumer_wallet.address, Web3.toWei(1, "ether"), {"from": publisher_wallet}
        )
    
        # consume
>       _ = ocean_assets.download_file(ddo.did, consumer_wallet)

tests/integration/ganache/test_consume_flow.py:154: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/enforce_typing/decorator.py:29: in wrapper
    return func(*args, **kwargs)
ocean_lib/ocean/ocean_assets.py:747: in download_file
    order_tx_id = self.pay_for_access_service(ddo, wallet)
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/enforce_typing/decorator.py:29: in wrapper
    return func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocean_lib.ocean.ocean_assets.OceanAssets object at 0x7f5efaee31f0>
ddo = <ocean_lib.assets.ddo.DDO object at 0x7f5efac8d9a0>
wallet = <LocalAccount '0x529043886F21D9bc1AE0feDb751e34265a246e47'>
service = <ocean_lib.services.service.Service object at 0x7f5efab6aaf0>
consume_market_order_fee_address = '0x529043886F21D9bc1AE0feDb751e34265a246e47'
consume_market_order_fee_token = '0x2473f4F7bf40ed9310838edFCA6262C17A59DF64'
consume_market_order_fee_amount = 0
consumer_address = '0x529043886F21D9bc1AE0feDb751e34265a246e47', userdata = None

    @enforce_types
    def pay_for_access_service(
        self,
        ddo: DDO,
        wallet,
        service: Optional[Service] = None,
        consume_market_order_fee_address: Optional[str] = None,
        consume_market_order_fee_token: Optional[str] = None,
        consume_market_order_fee_amount: Optional[int] = None,
        consumer_address: Optional[str] = None,
        userdata: Optional[dict] = None,
    ):
        # fill in good defaults as needed
        service = service or ddo.services[0]
        consume_market_order_fee_address = (
            consume_market_order_fee_address or wallet.address
        )
        consume_market_order_fee_amount = consume_market_order_fee_amount or 0
        if consume_market_order_fee_token is None:
            OCEAN_address = get_ocean_token_address(self._config_dict)
            consume_market_order_fee_token = OCEAN_address
        consumer_address = consumer_address or wallet.address
    
        # main work...
        dt = Datatoken(self._config_dict, service.datatoken)
        balance = dt.balanceOf(wallet.address)
    
        if balance < Web3.toWei(1, "ether"):
            raise InsufficientBalance(
                f"Your token balance {balance} {dt.symbol()} is not sufficient "
                f"to execute the requested service. This service "
                f"requires 1 wei."
            )
    
        consumable_result = is_consumable(
            ddo,
            service,
            {"type": "address", "value": wallet.address},
            userdata=userdata,
        )
        if consumable_result != ConsumableCodes.OK:
>           raise AssetNotConsumable(consumable_result)
E           ocean_lib.agreements.consumable.AssetNotConsumable: 2

ocean_lib/ocean/ocean_assets.py:824: AssetNotConsumable
---------------------------- Captured stdout setup -----------------------------

Transaction sent: 0x101e7c1c109cbe2fb5fa42fba6eef4f71d5e029dc0033dc5c0574c56315b44c5
  Gas price: 1e-09 gwei   Gas limit: 41437   Nonce: 245

  ERC20Template.mint confirmed   Block: 9[67](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:68)   Gas used: 37670 (90.91%)

----------------------------- Captured stdout call -----------------------------

Transaction sent: 0xc491fae8de315597e5d9c9b0a51f19181730[68](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:69)7e10318ee196bb694d749c8c56
  Gas price: 1e-09 gwei   Gas limit: 633853   Nonce: 446

  ERC721Factory.deployERC721Contract confirmed   Block: 968   Gas used: 562003 (88.66%)


Transaction sent: 0xf30f816f8bfc51b95401380ef51b68fc80badcf2cc4574bdae5daa3cbb780d4b
  Gas price: 1e-09 gwei   Gas limit: 516476   Nonce: 447

  ERC721Template.createERC20 confirmed   Block: 969   Gas used: 455034 (88.10%)


Transaction sent: 0x443fcb8e5f67592030001b6c4800798ab745e128b13f07b2597809e491e03b86
  Gas price: 1e-09 gwei   Gas limit: 176103   Nonce: 448

  ERC721Template.setMetaData confirmed   Block: 970   Gas used: 158302 (89.89%)


Transaction sent: 0x5a6388aa0c350a5d18e690c1495770fbae6b55aae3211107fcbffbaeaf6b4705
  Gas price: 1e-09 gwei   Gas limit: 76936   Nonce: 449

  ERC20Template.mint confirmed   Block: 971   Gas used: 69232 (89.99%)

Resolve did...
Order access...
----------------------------- Captured stderr call -----------------------------
2022-11-30 12:14:52 fv-az402-725 ocean[9952] INFO Successfully created NFT with address 0xc29C87F289800aE4B7bf1886A337707cc6a7d6F5.
2022-11-30 12:14:53 fv-az402-725 ocean[9952] INFO Successfully created datatoken with address 0xc6D5a4Cc46e51680D417959d725eBF05265F7dFb.
2022-11-30 12:14:53 fv-az402-725 ocean_lib.data_provider.data_encryptor[9952] INFO Asset urls encrypted successfully, encrypted urls str: 0x04bc1c3fccffc49e9b1b57c615d2c8d22ff0922cf83dbd1c62e7a7ae196669ea52377683da728f1594c416ffff2fcbfb4b0612dde61d8670874307b71b80fd1b0c0ec81738708a9297af7a5dfa8ae3649e8e68f406b2f60eac7269ca720aa21f05bd5facdf65bf4c733a2f296628b346d704db6fad7d71d1ecc2accf6f6ae1d495ce86300432ee76bbbb2057a9ec944f2aaf7ebc2be2baa35c8056dcc238fd941eab3c783c2e45934b4a8a481d4453a89e01943865b299fca4d2f73b714886f92a6533e91f89533798c662d020957461ae0b2859f9633b8a0eb0a5911c767f628f4a959292d095c5e9a7b46e16a0a949e7dbd2532075e303cce5f453c31d57b621e84ffdc2f63e49416703f6851696970f54baccd391a6370e96ae60f34b9d4f843026019c7de87b450d267663a13ff19691c846a66048c994bc7100d88a30, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
2022-11-30 12:14:53 fv-az402-725 ocean_lib.data_provider.data_encryptor[9952] INFO Asset urls encrypted successfully, encrypted urls str: 0x04f78b809a911d714def33d566a6784e3a965101c44cc463daca03b9209bf52df1bdbc3b195b88f67446c28ff3d4d1b3bbe1d129056588c48a8c0328ca75fbcb401614a86a6176744e054012e8c0f277ac24a1b5b0ef4449302c2e0a16935705219579e8f2b404eaf02cfc8ec6c932c88f3447a80599adb4452f730e840aa51e0649bde29cf6f1caf5a11423cfef5d943290f5f8ac278cbaf4c211216b37e828de4bde3b93db3d1ea5e03a09f4c839689ed620acd60a95ed10d7ae37bcf2148931131a5d74ab6c1bc490dd5d02037df51a8ebcfe77d526874923ac4d5bb072937f151802d513fe64695cce27b5cdfd9c17416e4e3757f71ebdc511b31ce3e6000a962d8c6fd2ba3c71a835694a3254ab468d0ec734a61441fb66f0202261073a2a81169a8858548e876e9c77479593bf71ac8bfc1dc41716b230eafa203ec25af5f29faba08bfff5c4e76c673f497b8e7efea000db61bfdd3cc1a31618a4d0984f077e5889c449b0b9f948d12a9cd61f05a5d2854489fb0f2238fd4329368c50e4f8ff6120b68416ec7987e2881be7dcffdc1ad353c81bb7516f96c4d146d06934295a7f6e7cb413d88e18e7078479fabad6fc2110c81e4e9e05566d6a09f0ff1aa1e4861b71a8d5a68f6c76701b80d89467e1afdc27832f323a323d6cc1a9e76f06fc8ed35ab252206078d2e336609b7dc715316b4c7cdd64f12396ea4b5f935413015f8c843c3eb0a64231889065c41d7ab6f8afefd0795e1360949c159b5de5f816d99c0e271f5904d5c2befd68a7ae1038d20b0c33041b085f57b5ba2009b76ddb74e5828e3e443acdac77770a9cca16c51db64247a36b96e206e6427e01b9e952d72807a314b73c06e1ef61a30dcdc56b891c34bf8ac725b1e6c59a4ce2c6b0202aaf9d529d257276fa0608de594c7e8e216bf4516a8cc91a70b770ee7dbc80187134d84691b2fabdd2529b4b3b99de617b4d5a180aa6cbb23aafc8876b4f8347f1714539dbc42d0648161a5caf0a0d284ce14ffc4303af43354a49ff0d3e260d7ea1b87ccb63b6f2e4e6548eb0d1eaff512d9e19e185818f3c0f39a8ecdc1cb507128e16f16d0a656e802db5b0f385f7452af74c61cb1d742a79232170100cc5d2e93ec0555e9b8da8e605f8764f706010da378a72f0ff2b4965ea096b0b2728b7562b1d656f9c77ab27c4ee104234a03c909ba15e59c45d4278c21b4f9ea6946f38532f4e45d62338c4b81f33aa2154d316b941c8b308c4c4fb40fae2c1ddd3483383a0a663acebf31a58c6bc57c950a17c996bc1adcaf67eedb776a63f03429ca9f809c39c5dcce61b0dbfa91721b271423786d1da9df95bdb9124999f1b28a48417797b170b013452a8d2ae51629778ec303a6a81510a2c17daa4109de1e3dc3b06b283383d1f07a2674f5fdcdf1582d7, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
------------------------------ Captured log call -------------------------------
INFO     ocean:ocean_assets.py:455 Successfully created NFT with address 0xc29C87F289800aE4B7bf1886A337707cc6a7d6F5.
INFO     ocean:ocean_assets.py:530 Successfully created datatoken with address 0xc6D5a4Cc46e51680D417959d725eBF05265F7dFb.
INFO     ocean_lib.data_provider.data_encryptor:data_encryptor.py:55 Asset urls encrypted successfully, encrypted urls str: 0x04bc1c3fccffc49e9b1b57c615d2c8d22ff0922cf83dbd1c62e7a7ae196669ea52377683da728f1594c416ffff2fcbfb4b0612dde61d8670874307b71b80fd1b0c0ec81738708a9297af7a5dfa8ae3649e8e68f406b2f60eac7269ca720aa21f05bd5facdf65bf4c733a2f296628b346d704db6fad7d71d1ecc2accf6f6ae1d495ce86300432ee76bbbb2057a9ec944f2aaf7ebc2be2baa35c8056dcc238fd941eab3c783c2e45934b4a8a481d4453a89e01943865b299fca4d2f73b714886f92a6533e91f89533798c662d020957461ae0b2859f9633b8a0eb0a5911c767f628f4a959292d095c5e9a7b46e16a0a949e7dbd2532075e303cce5f453c31d57b621e84ffdc2f63e49416703f6851696970f54baccd391a6370e96ae60f34b9d4f843026019c7de87b450d267663a13ff19691c846a66048c994bc7100d88a30, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
INFO     ocean_lib.data_provider.data_encryptor:data_encryptor.py:55 Asset urls encrypted successfully, encrypted urls str: 0x04f78b809a911d714def33d566a6784e3a965101c44cc463daca03b9209bf52df1bdbc3b195b88f67446c28ff3d4d1b3bbe1d129056588c48a8c0328ca75fbcb401614a86a6176744e054012e8c0f277ac24a1b5b0ef4449302c2e0a16935705219579e8f2b404eaf02cfc8ec6c932c88f3447a80599adb4452f730e840aa51e0649bde29cf6f1caf5a11423cfef5d943290f5f8ac278cbaf4c211216b37e828de4bde3b93db3d1ea5e03a09f4c839689ed620acd60a95ed10d7ae37bcf2148931131a5d74ab6c1bc490dd5d02037df51a8ebcfe77d526874923ac4d5bb072937f151802d513fe64695cce27b5cdfd9c17416e4e3757f71ebdc511b31ce3e6000a962d8c6fd2ba3c71a835694a3254ab468d0ec734a61441fb66f0202261073a2a81169a8858548e876e9c77479593bf71ac8bfc1dc41716b230eafa203ec25af5f29faba08bfff5c4e76c673f497b8e7efea000db61bfdd3cc1a31618a4d0984f077e5889c449b0b9f948d12a9cd61f05a5d2854489fb0f2238fd4329368c50e4f8ff6120b68416ec7987e2881be7dcffdc1ad353c81bb7516f96c4d146d06934295a7f6e7cb413d88e18e7078479fabad6fc2110c81e4e9e05566d6a09f0ff1aa1e4861b71a8d5a68f6c76701b80d89467e1afdc27832f323a323d6cc1a9e76f06fc8ed35ab252206078d2e336609b7dc715316b4c7cdd64f12396ea4b5f935413015f8c843c3eb0a64231889065c41d7ab6f8afefd0795e1360949c159b5de5f816d99c0e271f5904d5c2befd68a7ae1038d20b0c33041b085f57b5ba2009b76ddb74e5828e3e443acdac77770a9cca16c51db64247a36b96e206e6427e01b9e952d72807a314b73c06e1ef61a30dcdc56b891c34bf8ac725b1e6c59a4ce2c6b0202aaf9d529d257276fa0608de594c7e8e216bf4516a8cc91a70b770ee7dbc80187134d84691b2fabdd2529b4b3b99de617b4d5a180aa6cbb23aafc8876b4f8347f1714539dbc42d0648161a5caf0a0d284ce14ffc4303af43354a49ff0d3e260d7ea1b87ccb63b6f2e4e6548eb0d1eaff512d9e19e185818f3c0f39a8ecdc1cb507128e16f16d0a656e802db5b0f385f7452af74c61cb1d742a79232170100cc5d2e93ec0555e9b8da8e605f8764f706010da378a72f0ff2b4965ea096b0b2728b7562b1d656f9c77ab27c4ee104234a03c909ba15e59c45d4278c21b4f9ea6946f38532f4e45d62338c4b81f33aa2154d316b941c8b308c4c4fb40fae2c1ddd3483383a0a663acebf31a58c6bc57c950a17c996bc1adcaf67eedb776a63f03429ca9f809c39c5dcce61b0dbfa91721b271423786d1da9df95bdb9124999f1b28a48417797b170b013452a8d2ae51629778ec303a6a81510a2c17daa4109de1e3dc3b06b283383d1f07a2674f5fdcdf1582d7, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
_______________________ test_script_execution[script10] ________________________

script = PosixPath('/home/runner/work/ocean.py/ocean.py/tests/generated-readmes/test_datatoken-enterprise.py')
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f5efb722af0>

    @pytest.mark.parametrize("script", scripts)
    def test_script_execution(script, monkeypatch):
        # README generation command:
        # mkcodes --github --output tests/generated-readmes/test_{name}.{ext} READMEs
    
        if (
            "developers" in script.name
            or "df" in script.name
            or "publish-flow" in script.name
            or "data-nfts-and-datatokens-flow" in script.name
            or "c2d-flow-more-examples" in script.name
            or "parameters" in script.name
        ):
            # developers.md skipped because it does not have end-to-end Python snippets, just console
            # df.md -- ditto
            # data-nfts-and-datatokens-flow.md and publish-flow skipped because it they run as prerequisites for the others, so they are tested implicitly
            # c2d-flow-more-examples skipped because it can not be parsed separately from c2d-flow
            return
    
        if (
            "predict-eth" in script.name
            or "simple-remote" in script.name
            or "c2d-flow" in script.name
        ):
            # TODO: implement remote flows readme tests
            # C2D FLOW IS NOW REMOTE. Can we have a local one?
            return
    
        runs_with_prerequisites = [
            "c2d-flow",
            "dispenser-flow",
            "datatoken-enterprise",
            "marketplace-flow",
            "key-value-flow",
            "profile-nfts-flow",
            "consume-flow",
            "search-and-filter-assets",
        ]
    
        globs = {}
        for item in runs_with_prerequisites:
            if item in script.name:
                prerequisite = pathlib.Path(
                    __file__,
                    "..",
                    "..",
                    "generated-readmes/test_data-nfts-and-datatokens-flow.py",
                )
                result = runpy.run_path(str(prerequisite), run_name="__main__")
                for key in [
                    "os",
                    "config",
                    "ocean",
                    "alice_wallet",
                    "bob_wallet",
                    "data_nft",
                    "datatoken",
                ]:
                    globs[key] = result[key]
    
        runs_with_publish = [
            "consume-flow",
            "datatoken-enterprise",
        ]
        for item in runs_with_publish:
            if item in script.name:
                prerequisite = pathlib.Path(
                    __file__,
                    "..",
                    "..",
                    "generated-readmes/test_publish-flow.py",
                )
                result = runpy.run_path(
                    str(prerequisite), run_name="__main__", init_globals=globs
                )
                for key in [
                    "ddo",
                    "ZERO_ADDRESS",
                    # "did",
                    "metadata",
                    "url_file",
                ]:
                    globs[key] = result[key]
    
>       runpy.run_path(str(script), run_name="__main__", init_globals=globs)

tests/readmes/test_readmes.py:101: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/runpy.py:265: in run_path
    return _run_module_code(code, init_globals, run_name,
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/runpy.py:97: in _run_module_code
    _run_code(code, mod_globals, init_globals,
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/runpy.py:87: in _run_code
    exec(code, run_globals)
tests/generated-readmes/test_datatoken-enterprise.py:4: in <module>
    datatoken_enterprise_token = data_nft.create_datatoken(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocean_lib.models.data_nft.DataNFT object at 0x7f5efd26aca0>
name = 'DT1Name', symbol = 'DT1Symbol'
from_wallet = <LocalAccount '0x02354A1F160A3fd7ac8b02ee91F04104440B28E7'>
template_index = 2, minter = None, fee_manager = None
publish_market_order_fee_address = None, publish_market_order_fee_token = None
publish_market_order_fee_amount = 0, bytess = None
datatoken_cap = 50000000000000000000

    def create_datatoken(
        self,
        name: str,
        symbol: str,
        from_wallet,
        template_index: Optional[int] = 1,
        minter: Optional[str] = None,
        fee_manager: Optional[str] = None,
        publish_market_order_fee_address: Optional[str] = None,
        publish_market_order_fee_token: Optional[str] = None,
        publish_market_order_fee_amount: Optional[int] = 0,
        bytess: Optional[List[bytes]] = None,
        datatoken_cap: Optional[int] = None,
    ) -> Datatoken:
        initial_list = self.getTokensList()
    
        local_values = locals().copy()
        create_args = {
            lv_index: local_values[lv_index]
            for lv_index in [
                "template_index",
                "name",
                "symbol",
                "from_wallet",
                "minter",
                "fee_manager",
                "publish_market_order_fee_address",
                "publish_market_order_fee_token",
                "publish_market_order_fee_amount",
                "bytess",
            ]
        }
    
        for default_attribute in [
            "minter",
            "fee_manager",
            "publish_market_order_fee_address",
        ]:
            if create_args[default_attribute] is None:
                create_args[default_attribute] = from_wallet.address
    
        if publish_market_order_fee_token is None:
            create_args["publish_market_order_fee_token"] = ZERO_ADDRESS
    
        if bytess is None:
            create_args["bytess"] = [b""]
    
        if template_index == 2 and not datatoken_cap:
            raise Exception("Cap is needed for Datatoken Enterprise token deployment.")
    
        if template_index == 2:
            create_args["datatoken_cap"] = datatoken_cap
    
        create_args["transaction_parameters"] = {"from": create_args["from_wallet"]}
        create_args.pop("from_wallet")
    
        self.create_erc20(**create_args)
    
        new_elements = [
            item for item in self.getTokensList() if item not in initial_list
        ]
    
>       assert len(new_elements) == 1, "new data token has no address"
E       AssertionError: new data token has no address

ocean_lib/models/data_nft.py:168: AssertionError
---------------------------- Captured stdout setup -----------------------------

Transaction sent: 0xdfaab7058a8930791eaacdd62282c8a986099e480a3e9d014451fbcee6ada479
  Gas price: 1e-09 gwei   Gas limit: 41437   Nonce: 265

  ERC20Template.mint confirmed   Block: 1057   Gas used: 37670 (90.91%)

----------------------------- Captured stdout call -----------------------------

Transaction sent: 0xa9b7d31a339c943d131770493b4a7a85185ec847f64d12fa23fe7fe72396e98f
  Gas price: 1e-09 gwei   Gas limit: 633853   Nonce: 506

  ERC721Factory.deployERC721Contract confirmed   Block: 1058   Gas used: 562003 (88.66%)

Created data NFT. Its address is 0x823b0D3c174B1B3C6F528e88604e066CC65e11E8

Transaction sent: 0x3449ab80ed648d026252a9634cb8a0f25b82835618f54d09d938c92b21bb08bb
  Gas price: 1e-09 gwei   Gas limit: 516502   Nonce: 507

  ERC721Template.createERC20 confirmed   Block: 1059   Gas used: 455058 (88.10%)

Created datatoken. Its address is 0x7e4CE7ab7ebD9135f4705e0D515706921A3e0bFE
config.metadata_cache_uri = http://172.15.0.5:5000/
config.provider_url = http://172.15.0.4:8030
alice_wallet.address = '0x02354A1F160A3fd7ac8b02ee91F04104440B28E7'
data NFT name: NFT1
data NFT symbol: NFT1
datatoken name: Datatoken 1
datatoken symbol: DT1

Transaction sent: 0x508b4256f0e3179013a9871137dc1ad36bcb98e2a8e29c6db5e73b524eda9e9b
  Gas price: 1e-09 gwei   Gas limit: 634117   Nonce: 508

  ERC721Factory.deployERC721Contract confirmed   Block: 1060   Gas used: 562243 (88.67%)


Transaction sent: 0xcb9b1187724fcea7de8dd8e383200b2bb1d4d13f29fad4cb3468eff9384add2f
  Gas price: 1e-09 gwei   Gas limit: 516608   Nonce: 509

  ERC721Template.createERC20 confirmed   Block: 1061   Gas used: 455154 (88.10%)


Transaction sent: 0x5a2e47d1b4720fc52dc55d565e46b3b2c189dd0ab4d2f1a65287f92d86b0563a
  Gas price: 1e-09 gwei   Gas limit: 1761[69](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:70)   Nonce: 510

  ERC721Template.setMetaData confirmed   Block: 1062   Gas used: 158362 (89.89%)

Just published asset, with did=did:op:e119308a0139d77767c2a9a45d0c8[70](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:71)ba36f64bbbd280dc18b4dd27f10e2c703

Transaction sent: 0x630791ed686318c17e40f8189be3fd40c6d0bd0aee4fe152187bd2f8a50a51ce
  Gas price: 1e-09 gwei   Gas limit: 634117   Nonce: 511

  ERC721Factory.deployERC721Contract confirmed   Block: 1063   Gas used: 562243 (88.67%)


Transaction sent: 0xf5634dc47e658f775ff0f7df9f0ec4f79372f9421e4582f8efad4126f02d6bce
  Gas price: 1e-09 gwei   Gas limit: 516502   Nonce: 512

  ERC721Template.createERC20 confirmed   Block: 1064   Gas used: 455058 (88.10%)


Transaction sent: 0xd3685ccbd41f851caaa7f72401780593b9517ed7531ae9947a9c09cbb0ec85d1
  Gas price: 1e-09 gwei   Gas limit: 176209   Nonce: 513

  ERC721Template.setMetaData confirmed   Block: 1065   Gas used: 158398 (89.89%)

Just published asset, with did=did:op:9aee4d2af2e3ad019a3b4da334d74dfe42aa0ede0fa7fcc7052f9963d0ceb2cd

Transaction sent: 0xd1aa86ba6aaa410203df0c6217e773efadcf7a355e6e32baf2ebd1760cb0f66e
  Gas price: 1e-09 gwei   Gas limit: 482824   Nonce: 514

  ERC721Template.createERC20 confirmed   Block: 1066   Gas used: 420[71](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:72)1 (87.14%)

----------------------------- Captured stderr call -----------------------------
2022-11-30 12:23:59 fv-az402-[72](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:73)5 ocean[9952] INFO Successfully created NFT with address 0x96aaFb79d301a5D5f8Df1169106041D9CaBEb813.
2022-11-30 12:24:03 fv-az402-725 ocean[9952] INFO Successfully created datatoken with address 0xCCdCefFd85b405aF3E0C[73](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:74)bFe67B1Da23954b31C.
2022-11-30 12:24:04 fv-az402-725 ocean_lib.data_provider.data_encryptor[9952] INFO Asset urls encrypted successfully, encrypted urls str: 0x042c0dfaf65901259d2947c72f9493a207979a3edc7129aeb98e5bceea5fcc60bafb4348f92ca4559022543a3201c667f2564c9fc0e4ee863462bb1dc6059a3918bc42373a3f492282a[74](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:75)629ad67723f274242a41f74f92fe0e59d8e84201bd6d7f63c6555fe851cd1d3622a5a82a8096b9b8603ee5cea5a33ee78d08809b2f330a72714ab3aa256ec039a46d4630f992109cb228d2b2b35dccb627c8b79f08cf85b0644fe677d705a89bc3d7691ac80a1ab737b683b497e4c1970edbea726032368c0e0d1bfc1903beb1bd17d20e5d86d76f9ddb7817213cc1e48eddf3abbe61151372a2ce69acfac72555c6afee61202f8133b5c7d7a32f2dac9f68fbb95bc5971cab64894283e1e2971f6c7aee170c3499c973e919eca0169eb21d924ff8fec6c5204d3bf9c43be7f27102c77cd9ba934c3b09c6481a56d0b52ca28baf6, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
2022-11-30 12:24:04 fv-az402-725 ocean_lib.data_provider.data_encryptor[9952] INFO Asset urls encrypted successfully, encrypted urls str: 0x04a253dc5847f6233ef3566b015dd3e[75](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:76)d0a354045ebfb09b23e9f8fe7cced3df5c3f531041b9c1d7c6d990cf44db9e6a83b400c34f541c7e6fc13a73b65664d7489331ec4e03b29bc173bf3a01feb58036dca03b60bd1a0420e2c3d7e652018188f46efba5b844e75673c4f7982e09a7c2604823cdabcf96e7818fac396da637d98aa5036a52e5a5956e6acb62b78cc9da19e8aa57dc9aa1f95229339ab30a9ed212cfc84cd2c060b8f41c9efe69ec1866d74a3c850aba11174bab892d5ce064ba20fe88d47a84608625cb6ea8f0b9ec414d00067ccb3f52b7446198c87d1c05a7a0ff22fdbd2807e723c6235be2286fa367817e032970e6a8f5[76](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:77)d611253f881c68d400d8f82cd2f327e12b4c9f017f78a16c9e10dc81eacda2d3c14dfc19d820fe78cb2e59bc488a7a9b8f27eb264d8a98336f09be55442997d1213318e2b919605883c5192cf4e506f[77](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:78)cd3d9183ab22ea1b40081fd2d97d9c88735c7601692ab69917200d5f8ad19146cdeb3d0d9457aa2533aa09373ac59f47a1e3eab8cf5c6bee2985ca43644b55c8300be5db989cc28123bda3d65a1d6446c0a4a2e9dd04689c690e570138912f479723045b0597980e9cd0[78](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:79)51d49393523fcfc723fe4460aa3da0892b18a8624973cefd453f52d22917e88298733a9af5676120ab41e56804cd166da0b00d4771a84066018c6985eb1e002235759c233a3a0a03df97896f6707f52e4805ac9607e2b0f060b3e0b2b32ab1[79](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:80)ef7e5af98af4a75d26f4a46943a6f502b2ae8cd059a0ac3d6e8f708588193d457118d8ddb21db6c559b169852c6e6217c61cf7fcb5dc831cefd394fb2ab6211df35ae9bad28329257b230164b55adc4ac30e98fe3b0b53d2d379c6fa8b8ff4f2c977d8183b3c72c873e84651ba2e73830071be065b28a6b974a4235019ce2f250fe3c7027439d319cfdbd35285eb6260e378e59aad7981bdf783fa9b638207232b06d5c611ffd5de107793a2bf46471c56ecda06225c23f0db069500a11a5e0501f18472c49392121a0cdb5833d7d3d79aaf3ae0569438ce2fe458c4683cf5fb410178f1c1f3b3e6ff3765f0b1933d21b218db56453c585e924a8164316d388ecdfe57d45e584196f9b0cd418cd7c0b9c39d6d47064ffb71a4b07513638533797506398c4d5b12bac364c5d4c7b3b8141bddcafc9527dba23b71d7c1567e4ba62a73cd41709b01b67234f2b774e7d2d24145ca48e59363a1a5ec544685bcf393c1349199750b3d5e12c3c63386fca1b0cae5a2b7a9c78f73cb21c0ab1b55f57a3985a2[80](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:81)43d242125c6e36a17ed1f6cd414b0d7fd3edd15315ee3bda156586694891491ea77144ecc54d93d1a8d261a484729c951b9e40946ed7ec0070f2d55079426c34368bc6[81](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:82)6246564d35ab62ff1b9c654f5321680b0[82](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:83)365852d9f815870d2a0e1f645, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
2022-11-30 12:24:20 fv-az402-725 ocean[9952] INFO Successfully created NFT with address 0x9E1c7D1fBE8B7B63E77fdfCbEee53E6F2dBC6f12.
2022-11-30 12:24:20 fv-az402-725 ocean[9952] INFO Successfully created datatoken with address 0x80CBa3910Ab4ff8953c3589809D61cfD34245F13.
2022-11-30 12:24:21 fv-az402-725 ocean_lib.data_provider.data_encryptor[9952] INFO Asset urls encrypted successfully, encrypted urls str: 0x04a8c3fbe71136fe5140e7eb364604ebaa2a33182ccb52fd7c8664fc445bbf32fa78a49133a45f0b2d4d9fb792b386[83](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:84)0b89c2cd93423199129e8e828cd2b9fa35b60dc59aaa6b26fa0aac[84](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:85)acc2d9e4550d48e1ed45270f956e5bf2bbdd7490899a73039bd2ccdf2ee14749378cc89646d5d9d17067afa380f7e8e3fa07377cb40328cbab4c1ca278d19887c315661225d3dbfea4a4246010fb128b3d9c2e78cf9da4ae4b9d28ec556a97c915fac90c6a35266737f7d26ea18f21bc8cdce24cc8e5fd1edadebaa84e51dd570da53df6fd8e5c20f52c8ca3c7e6ec74a15f9102cbcdce651bd6f46901783977a619c5bd1a33ec01bf515a60ea50e15ed00e8e10916f565cbedf7980e35075616778954442266935b191fc6dd80bba3e0e3f8b9fb766c1cd432420da78912e6ace6bdb531bf60c3c6a9dd7fd4c14cef4663809, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
2022-11-30 12:24:21 fv-az402-725 ocean_lib.data_provider.data_encryptor[9952] INFO Asset urls encrypted successfully, encrypted urls str: 0x045[85](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:86)424382258e51c3029b39e88cf83d0ce113dff60f3565b5fa2b7c7c39384f5d98211b13dbfe5eb40b08e65364b8b9d2fe1291acc44a8034b10fdd4d42c81121b83c10450833b382a31f648b0fc9b1216604c1cdfa57cfd79785e[86](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:87)7c0b458c3e31576e3ed92a893b83f0e02bc50e08b3d7ba15a161f9b1e7a33119e[87](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:88)90266c391519065a6fdfc9031b24d630478ef56dcd8f8c92a917d3be2a080e1aab349af669f0a517fe14bc562fca93995c973640e8f267d791350465699b67b703cee607498cd865cb4eff2b4a79b0b19f5c15400235b2c428be0dc7a1abfa4f0b0e76005c4177dafd84480b0f2a35be6fcf084ea99ee1ed301ba401e51af21970354b9139d18d8c8d326fa5476f21edb4b3dc63ca01ef16a7c0bc4790435987e0d797c5a2[88](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:89)a9478dcdfb8309f8a8b83432744cef81790df251fc2ab516b65c33d0cfd7bff0e8ce15ab03db4f7dab91aaa553c3ec08e224074733242ca52380a8a7bbd0235ab14f51696e923afb6a73266259f40699d86d99f3bab92e6b36306580395b6457633ca5dca5614c26ae78b25b799b8735a1e829566f5cbc9ee79dcdce2d3906a411ced60dd0a74fe1d3dc9f77be391b34b42abff5e504bb97a3524e73ebba2c1cb8376c2c953425b04e41c8e29f[89](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:90)95d7c9dd7bc96f9962623140e52f6bd554ed75dea9982f3b475a436de53ea86bb5e1d5cc5b1badbb30314a703e98f8ed2142375e3edb1deda9cd8f9ad2a05d825ca6bc7b80eb31dd29d4960e05c168073df3da10175cd09bc54088f23de67da6f3008f00bb0687cfe7222e2eb83d152f20cec6b26d430e6faf61d43f0b9c87b41841eeac528dca5a9a01c85368c5fe8e2b3110a54039ef0897ebcf76ec0dbdbfaa1f7b2f7417862e033787a671dfa68a980f0d85c08bfe882825d86f215921eb1195edf26910f36ed982c5bb95f2ebb9b43[90](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:91)b42d80a9c99207cb11a9a2e68aa7d54d4940729ae0fb44bc0d32c7bdab5ebc5a259ddba90628135037e7220e7eb0c44ee12dd[91](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:92)ce7e524c17a1df1c7b90547c9eb70634c9ae344fab163cbac9ce662f3e271ee68720a7fb44d[92](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:93)c46eba295f10981048d3c362eadfe8024339cd5666042af9a740a2ee3fb6d5409d13cebbb30c8c9d9ef5ce95498a9b5f7ad463cafe01b254ac4adf5fef45[93](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:94)cc9eea6c9c9e64a2acb1bd0fcad4fcc0df366636fb732e0d00493d79c2b3ff0304e7cd11edf5e2d881b461656a7c59aa5aa7464af1c27934e3abe034b196a55f6b2b98f95f66843ac8316d6f817185ab770352fc8d001ecfe9d191761d328a04efa623cfa2824d7b67f4169db99ca729772e68061734ea0f77a68b7bc87238deba92986b064a25504e9a7[94](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:95)b26be23be578ee360e8b6139ce53d8b9ed582f34fe7dcfeaf110c6daf6ce216b0feea7efe67719ef59d4b9eda722ad3fc8f76d1, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
------------------------------ Captured log call -------------------------------
INFO     ocean:ocean_assets.py:455 Successfully created NFT with address 0x96aaFb79d301a5D5f8Df1169106041D9CaBEb813.
INFO     ocean:ocean_assets.py:530 Successfully created datatoken with address 0xCCdCefFd85b405aF3E0C73bFe67B1Da23[95](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:96)4b31C.
INFO     ocean_lib.data_provider.data_encryptor:data_encryptor.py:55 Asset urls encrypted successfully, encrypted urls str: 0x042c0dfaf65901259d2947c72f9493a207979a3edc7129aeb98e5bceea5fcc60bafb4348f92ca4559022543a3201c667f2564c9fc0e4ee863462bb1dc6059a3918bc42373a3f492282a74629ad67723f274242a41f74f92fe0e59d8e84201bd6d7f63c6555fe851cd1d3622a5a82a80[96](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:97)b9b8603ee5cea5a33ee78d08809b2f330a72714ab3aa256ec039a46d4630f992109cb228d2b2b35dccb627c8b79f08cf85b0644fe677d705a89bc3d7691ac80a1ab737b683b4[97](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:98)e4c1970edbea726032368c0e0d1bfc1903beb1bd17d20e5d86d76f9ddb7817213cc1e48eddf3abbe61151372a2ce69acfac72555c6afee61202f8133b5c7d7a32f2dac9f68fbb95bc5971cab64894283e1e2971f6c7aee170c3499c973e919eca0169eb21d924ff8fec6c5204d3bf9c43be7f27102c77cd9ba934c3b09c6481a56d0b52ca28baf6, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
INFO     ocean_lib.data_provider.data_encryptor:data_encryptor.py:55 Asset urls encrypted successfully, encrypted urls str: 0x04a253dc5847f6233ef3566b015dd3e75d0a354045ebfb09b23e9f8fe7cced3df5c3f531041b9c1d7c6d990cf44db9e6a83b400c34f541c7e6fc13a73b65664d7489331ec4e03b29bc173bf3a01feb58036dca03b60bd1a0420e2c3d7e652018188f46efba5b844e75673c4f7[98](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:99)2e09a7c2604823cdabcf96e7818fac396da637d98aa5036a52e5a5956e6acb62b78cc9da19e8aa57dc9aa1f95229339ab30a9ed212cfc84cd2c060b8f41c9efe69ec1866d74a3c850aba11174bab892d5ce064ba20fe88d47a84608625cb6ea8f0b9ec414d00067ccb3f52b7446198c87d1c05a7a0ff22fdbd2807e723c6235be2286fa367817e032970e6a8f576d611253f881c68d400d8f82cd2f327e12b4c9f017f78a16c9e10dc81eacda2d3c14dfc19d820fe78cb2e59bc488a7a9b8f27eb264d8a98336f09be55442[99](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:100)7d1213318e2b919605883c5192cf4e506f77cd3d9183ab22ea1b40081fd2d97d9c88735c7601692ab69917200d5f8ad19146cdeb3d0d9457aa2533aa09373ac59f47a1e3eab8cf5c6bee2985ca43644b55c8300be5db989cc28123bda3d65a1d6446c0a4a2e9dd04689c690e570138912f479723045b0597980e9cd07851d49393523fcfc723fe4460aa3da0892b18a8624973cefd453f52d22917e88298733a9af5676120ab41e56804cd166da0b00d4771a84066018c6985eb1e002235759c233a3a0a03df97896f6707f52e4805ac9607e2b0f060b3e0b2b32ab179ef7e5af98af4a75d26f4a46943a6f502b2ae8cd059a0ac3d6e8f708588193d457118d8ddb21db6c559b169852c6e6217c61cf7fcb5dc831cefd394fb2ab6211df35ae9bad28329257b230164b55adc4ac30e98fe3b0b53d2d379c6fa8b8ff4f2c977d8183b3c72c873e84651ba2e73830071be065b28a6b974a4235019ce2f250fe3c7027439d319cfdbd35285eb6260e378e59aad7981bdf783fa9b638207232b06d5c611ffd5de107793a2bf46471c56ecda06225c23f0db069500a11a5e0501f18472c49392121a0cdb5833d7d3d79aaf3ae0569438ce2fe458c4683cf5fb4[101](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:102)78f1c1f3b3e6ff3765f0b1933d21b218db56453c585e924a8164316d388ecdfe57d45e584196f9b0cd418cd7c0b9c39d6d47064ffb71a4b07513638533797506398c4d5b12bac364c5d4c7b3b8141bddcafc9527dba23b71d7c1567e4ba62a73cd41709b01b67234f2b774e7d2d24145ca48e59363a1a5ec544685bcf393c1349199750b3d5e12c3c63386fca1b0cae5a2b7a9c78f73cb21c0ab1b55f57a3985a28043d242125c6e36a17ed1f6cd414b0d7fd3edd15315ee3bda156586694891491ea77144ecc54d93d1a8d261a484729c951b9e40946ed7ec0070f2d55079426c34368bc6816246564d35ab62ff1b9c654f5321680b082365852d9f815870d2a0e1f645, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
INFO     ocean:ocean_assets.py:455 Successfully created NFT with address 0x9E1c7D1fBE8B7B63E77fdfCbEee53E6F2dBC6f12.
INFO     ocean:ocean_assets.py:530 Successfully created datatoken with address 0x80CBa3910Ab4ff8953c3589809D61cfD34245F13.
INFO     ocean_lib.data_provider.data_encryptor:data_encryptor.py:55 Asset urls encrypted successfully, encrypted urls str: 0x04a8c3fbe71136fe5140e7eb364604ebaa2a33182ccb52fd7c8664fc445bbf32fa78a49133a45f0b2d4d9fb792b386830b89c2cd93423199129e8e828cd2b9fa35b60dc59aaa6b26fa0aac84acc2d9e4550d48e1ed45270f956e5bf2bbdd7490899a73039bd2ccdf2ee14749378cc89646d5d9d17067afa380f7e8e3fa07377cb40328cbab4c1ca278d19887c315661225d3dbfea4a4246010fb128b3d9c2e78cf9da4ae4b9d28ec556a97c915fac90c6a35266737f7d26ea18f21bc8cdce24cc8e5fd1edadebaa84e51dd570da53df6fd8e5c20f52c8ca3c7e6ec74a15f9[102](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:103)cbcdce651bd6f46901783977a619c5bd1a33ec01bf515a60ea50e15ed00e8e10916f565cbedf7980e35075616778954442266935b191fc6dd80bba3e0e3f8b9fb766c1cd432420da78912e6ace6bdb531bf60c3c6a9dd7fd4c14cef4663809, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
INFO     ocean_lib.data_provider.data_encryptor:data_encryptor.py:55 Asset urls encrypted successfully, encrypted urls str: 0x04585424382258e51c3029b39e88cf83d0ce113dff60f3565b5fa2b7c7c39384f5d98211b13dbfe5eb40b08e65364b8b9d2fe1291acc44a8034b10fdd4d42c81121b83c[104](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:105)50833b382a31f648b0fc9b1216604c1cdfa57cfd79785e867c0b458c3e31576e3ed92a893b83f0e02bc50e08b3d7ba15a161f9b1e7a33119e8790266c391519065a6fdfc9031b24d630478ef56dcd8f8c92a917d3be2a080e1aab349af669f0a517fe14bc562fca93995c973640e8f267d791350465699b67b703cee607498cd865cb4eff2b4a79b0b19f5c15400235b2c428be0dc7a1abfa4f0b0e76005c4177dafd84480b0f2a35be6fcf084ea99ee1ed301ba401e51af21970354b9139d18d8c8d326fa5476f21edb4b3dc63ca01ef16a7c0bc4790435987e0d797c5a288a9478dcdfb8309f8a8b83432744cef81790df251fc2ab516b65c33d0cfd7bff0e8ce15ab03db4f7dab91aaa553c3ec08e224074733242ca52380a8a7bbd0235ab14f51696e923afb6a73266259f40699d86d99f3bab92e6b36306580395b6457633ca5dca5614c26ae78b25b799b8735a1e829566f5cbc9ee79dcdce2d3906a411ced60dd0a74fe1d3dc9f77be391b34b42abff5e504bb97a3524e73ebba2c1cb8376c2c953425b04e41c8e29f8995d7c9dd7bc96f9962623140e52f6bd554ed75dea9982f3b475a436de53ea86bb5e1d5cc5b1badbb30314a703e98f8ed2142375e3edb1deda9cd8f9ad2a05d825ca6bc7b80eb31dd29d4960e05c168073df3da10175cd09bc54088f23de67da6f3008f00bb0687cfe7222e2eb83d152f20cec6b26d430e6faf61d43f0b9c87b41841eeac528dca5a9a01c85368c5fe8e2b3110a54039ef0897ebcf76ec0dbdbfaa1f7b2f7417862e033787a671dfa68a980f0d85c08bfe882825d86f215921eb1195edf26910f36ed982c5bb95f2ebb9b4390b42d80a9c99207cb11a9a2e68aa7d54d4940729ae0fb44bc0d32c7bdab5ebc5a259ddba90628135037e7220e7eb0c44ee12dd91ce7e524c17a1df1c7b90547c9eb70634c9ae344fab163cbac9ce662f3e271ee68720a7fb44d92c46eba295f[109](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:110)81048d3c362eadfe8024339cd5666042af9a740a2ee3fb6d5409d13cebbb30c8c9d9ef5ce95498a9b5f7ad463cafe01b254ac4adf5fef4593cc9eea6c9c9e64a2acb1bd0fcad4fcc0df366636fb732e0d00493d79c2b3ff0304e7cd11edf5e2d881b461656a7c59aa5aa7464af1c27934e3abe034b196a55f6b2b98f95f66843ac8316d6f817185ab770352fc8d001ecfe9d191761d328a04efa623cfa2824d7b67f4169db99ca729772e68061734ea0f77a68b7bc87238deba92986b064a25504e9a794b26be23be578ee360e8b6139ce53d8b9ed582f34fe7dcfeaf[110](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:111)c6daf6ce216b0feea7efe67719ef59d4b9eda722ad3fc8f76d1, encryptedEndpoint http://172.15.0.4:8030/api/services/encrypt
=============================== warnings summary ===============================
ocean_lib/aquarius/test/test_aquarius.py::test_aqua_functions_for_single_ddo
  /opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/brownie/network/event.py:490: UserWarning: 0x1D5CEbD46Eb0a[120](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:121)13DbE1A66733AD39d[410](https://github.com/oceanprotocol/ocean.py/actions/runs/3582991399/jobs/6027874189#step:7:411)F061: Event log does not contain enough topics for the given ABI - this is usually because an event argument is not marked as indexed
    warnings.warn(f"{address}: {exc}")
...
```

--- a/tests/integration/ganache/test_consume_flow.py
+++ b/tests/integration/ganache/test_consume_flow.py
@@ -19,7 +19,6 @@ from tests.resources.ddo_helpers import get_first_service_by_type
 
 
 @pytest.mark.integration
-@pytest.mark.skip(reason="Don't skip, once fixed #1137")
 def test_consume_flow(
     config: dict,
     publisher_wallet,
@@ -140,8 +139,8 @@ def test_compact_publish_and_consume(
     ocean_assets = OceanAssets(config, data_provider)
 
     # publish
-    name = "CEXA"
-    url = "https://cexa.oceanprotocol.io/ohlc?exchange=binance&pair=ETH/USDT"
+    name = "My asset"
+    url = "https://raw.githubusercontent.com/trentmc/branin/main/branin.arff"
     (data_nft, datatoken, ddo) = ocean_assets.create_url_asset(
         name, url, publisher_wallet
     )

--- a/tests/integration/ganache/test_issue981_consume.py
+++ b/tests/integration/ganache/test_issue981_consume.py
@@ -26,8 +26,8 @@ def test1(
     ocean = Ocean(config)
 
     # Publish
-    url = "https://cexa.oceanprotocol.io/ohlc?exchange=binance&pair=ETH/USDT"
-    name = "CEXA ETH-USDT"
+    url = "https://raw.githubusercontent.com/trentmc/branin/main/branin.arff"
+    name = "My asset"
     (data_nft, datatoken, ddo) = ocean.assets.create_url_asset(
         name, url, publisher_wallet
     )

